### ISSUE TITLE
Incorrect background aspect ratio when playing in 16:10

### ISSUE THREAD

**Issue by SimonAfek** [07th-mod/meakashi#59]
**The problem**: In a certain scene in Meakashi Ch. 6, the background is incorrectly cropped when playing fullscreen on a 16:10 monitor (see screenshot). Normally, the game is fully letterboxed into a 16:9 aspect ratio, but for this one scene, the background image extends into where the letterboxing would normally be - from the cutoffs of the character sprite, the issue is obvious.

**The mod variant**: Full

**I bought the game at**: Mangagamer

**My operating system is**: Windows

**I installed the patch**: Using the Installer

**Mod Install Date**: Less than a week ago

![meakashi_16-10_clipping](https://user-images.githubusercontent.com/18552155/88730228-ec878e00-d135-11ea-9149-15600ee83bab.png)

**Comment by orian34** [comment]
It's the text box the issue here though?

**Comment by SimonAfek** [comment]
No, the text box is right where it's supposed to be - since I'm playing fullscreen on a 16:10 monitor, the game is normally letterboxed. Like this (notice the black bars on top and bottom):

![meakashi_16-10_letterbox](https://user-images.githubusercontent.com/18552155/88735961-d0392080-d138-11ea-9076-4de3bc8a4495.png)

**Comment by deepFlaw** [comment]
May be worth noting that the menu vanishing also happens while in NVL mode (with original art - so I saw no cutoff sprite), and I found that I couldn't save during that part either. I have not yet looked at the script to see if there's a clear reason for this and/or if it's intentional, but will take a look later today.

**Comment by tellowkrinkle** [comment]
The issue is that the game doesn't actually letterbox, it just puts things in the places you would expect for 16:9, and since most of the source images are also 16:9 there's never anything outside the 16:9 area.  Except when a source image isn't 16:9 for whatever reason.  Another example of this issue is in Onikakushi when they do the scrolling hinamizawa scene near the beginning, it fills the screen instead of being 16:9

Think of it like those old flash animations that would render at any aspect ratio but then you could see all the things that were supposed to be offscreen
